.gitlab/container_build/docker_linux.yml

```diff
@@ -260,6 +260,46 @@ docker_build_cluster_agent_arm64:
     - mv -vf $CWS_INSTRUMENTATION_BINARIES_DIR $BUILD_CONTEXT/
     - mv -vf Dockerfiles/agent/nosys-seccomp Dockerfiles/cluster-agent/
 
+docker_build_cluster_agent_fips_amd64:
+  extends: [.docker_build_job_definition_amd64]
+  rules: !reference [.on_tag_or_a7]
+  needs:
+    - job: cluster_agent_fips-build_amd64
+      artifacts: true
+    - job: cws_instrumentation-build_amd64
+      artifacts: true
+    - job: cws_instrumentation-build_arm64
+      artifacts: true
+  variables:
+    IMAGE: registry.ddbuild.io/ci/datadog-agent/cluster-agent
+    BUILD_CONTEXT: Dockerfiles/cluster-agent
+    TAG_SUFFIX: -fips
+  before_script:
+    - mv -vf $CLUSTER_AGENT_BINARIES_DIR/datadog-cluster-agent $BUILD_CONTEXT/
+    - mv -vf Dockerfiles/agent/nosys-seccomp Dockerfiles/cluster-agent/
+    - mv -vf $CWS_INSTRUMENTATION_BINARIES_DIR $BUILD_CONTEXT/
+    - nm --defined-only ${BUILD_CONTEXT}/datadog-cluster-agent | grep boringssl_fips_self_test
+
+docker_build_cluster_agent_fips_arm64:
+  extends: [.docker_build_job_definition_arm64]
+  rules: !reference [.on_tag_or_a7]
+  needs:
+    - job: cluster_agent_fips-build_arm64
+      artifacts: true
+    - job: cws_instrumentation-build_amd64
+      artifacts: true
```

Comment:
Any idea why we need to wait on the amd64 version of the job here? Same question but for arm64 on the other job.

Reply:
If I understood correctly, the DCA needs both arch to be able to provide the cws_instrumentation binary to any node running in the monitored cluster, regardless of the arch running the DCA